### PR TITLE
revised content of markdown files

### PR DESCRIPTION
**Pull request by Incei**

### CODE REVIEW

**Comment by Incei** [comment]
I somehow cannot add AndreDlm as a reviewer... ( @AndreDlm)

File: Input/Content/Translation.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ All pediatric translations are pure retrospective predictions, no pediatric phar
 
 The models were evaluated by ratio plots of area under the plasma concentration-time curve (AUC), or clearance (CL) values resulting from our predictions to the values observed during clinical studies, and by comparison of concentration-time profiles if available. As a quantitative measure of the descriptive and predictive performance of each model, the geometric mean fold error was calculated according to Eq. 1:
 
-Eq. 1: GMFE=10^((Σ|log10(pred PK parameter∕obs PK parameter)|)∕n)
+Eq. 1: $$ {\Huge  GMFE = 10^{\frac{\sum(|log(\frac{Pred}{Obs})|)}{n}} }  $$
```
Comment (Yuri05) [inline Input/Content/Translation.md:7]: replace $$ with $ (and no blanks after the opening $ and before the closing $):
`Eq. 1: ${\Huge  GMFE = 10^{\frac{\sum(|log(\frac{Pred}{Obs})|)}{n}} }$`
Eq. 1: ${\Huge  GMFE = 10^{\frac{\sum(|log(\frac{Pred}{Obs})|)}{n}} }$

**Comment by Yuri05** [comment]
@Incei comment out all inactivatetask lines in Workflow.R
Current workflow would fail on a new machine 
https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_P-gp_Ontogeny/blob/f08fcac54ab10fd91c650436c9fb65f62394c8a3/Workflow.R#L144-L151

**Review by Yuri05 — CHANGES_REQUESTED** [review]
s. comments

**Pull request merged**